synapse/lib/view.py

```diff
@@ -227,13 +236,36 @@ def pack(self):
 
         await synt.worker(runStorm())
 
+        necount = 0
+        editformat = opts.get('editformat', 'nodeedits')
+
         while True:
 
             mesg = await chan.get()
+            kind = mesg[0]
+
```

Comment:
Would we want to maybe short circuit "node" type messages here to speed them along?

Reply:
ok